lib/MusicBrainz/Server/Data/EditNote.pm

```diff
@@ -106,6 +113,27 @@ sub add_note
     }
 }
 
+sub find_by_recipient {
+    my ($self, $recipient_id, $limit, $offset) = @_;
+
+    my $query = <<'EOSQL';
+        SELECT editor, edit, text, post_time
```

Comment:
I’d suggest using `$self->_columns`. Or is there a special reason why you left out `id` here? Seems likely to me that the missing ID will cause issues down the line (e.g. isn’t there an `#anchor` on edit pages that contains the note ID?). – Also `$self->_table` below, probably, although changing the table name is less likely.